### ISSUE TITLE
Should add `rexml` gem to explicit dependency.

### ISSUE THREAD

**Issue by r-plus** [sds/slim-lint#182]
rubocop v1.66 remove `rexml` from dependency.
https://github.com/rubocop/rubocop/releases/tag/v1.66.0

slim_lint depends on `rexml` gem but that is not explicitly made a dependency, so it is ~not installed~ (Ruby 3.3 and bundler combination required explicitly dependent to bundled gem) and an error occurs.

https://github.com/sds/slim-lint/blob/c2b8e7af34075a1befaf2a22e0655340ba1a4707/lib/slim_lint/reporter/checkstyle_reporter.rb#L3

**Comment by r-plus** [comment]
@sds Hi.
This is error stack trace when execute slim_lint.

```
/opt/hostedtoolcache/Ruby/3.3.4/x64/lib/ruby/3.3.0/bundled_gems.rb:74:in `require': cannot load such file -- rexml/document (LoadError)
	from /opt/hostedtoolcache/Ruby/3.3.4/x64/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
	from /home/runner/work/make-server/make-server/vendor/bundle/ruby/3.3.0/gems/slim_lint-0.27.0/lib/slim_lint/reporter/checkstyle_reporter.rb:3:in `<top (required)>'
```

**Comment by sds** [comment]
Thanks for the report. Released in [0.29.0](https://github.com/sds/slim-lint/releases/tag/v0.29.0).
